### DISCUSSION TITLE
Draw Route Between Current Position and Selected Position

### DISCUSSION THREAD

**Discussion by azimazman** [Mapsui/Mapsui#1111] [Q&A] [answered]
Hello, Im newbie in programming about mapping. I need to build sample map application that track current position and user can click on any map to draw route using xamarin forms. I have done to get current position and view the map. But my issue is to draw the route. Can someone suggest me what method to be do to get it ?

**Comment by pauldendulk** [comment, marked as answer]
That is not supported out of the box. Perhaps you could use a library like itinero https://github.com/itinero/routing.

**Reply by Vincent1099** [reply]
hey, did solve it, so MapsUi has a similar feature?
